config/formats.js

```diff
@@ -3431,6 +3431,20 @@ exports.Formats = [
 					this.add('-anim', source, "Tail Slap", target);
 				};
 			}
+			if (move.id === 'hypervoice' && name === 'bumbadadabum') {
+				move.name = 'Open Source Software';
+				move.type = 'Electric';
+				move.basePower = 110;
+				move.accuracy = 95;
+				move.secondaries = [{chance:20, self:{boosts:{spa:-1}}, volatileStatus:'disable'}];
+				move.onTryHit = function (target, source) {
+					this.attrLastMove('[still]');
+					this.add('-anim', source, "Dark Void", target);
+				};
+				move.onHit = function () {
+					this.add('c|%Bumbadadabum|I\'d just like to interject for a moment. What you\'re referring to as Linux, is in fact, GNU/Linux, or as I\'ve recently taken to calling it, GNU plus Linux. Linux is not an operating system unto itself, but rather another free component of a fully functioning GNU system made useful by the GNU corelibs, shell utilities and vital system components comprising a full OS as defined by POSIX.');
```

Comment:
@bumbadadabum (you are totally a moderator now, nah, I just refer to you on GitHub)

I find it interesting that you call it GNU/Linux ([how it should be called according to GNU](https://www.gnu.org/gnu/why-gnu-linux.html)). At the same time, your signature move name "Open Source Software" ([when according to GNU it should have been Free Software](https://www.gnu.org/philosophy/open-source-misses-the-point.html)). Not really consistent.

Although, I guess it's just a political issue. Obviously nothing affecting merging this pull request.